docs/theming.md

```diff
@@ -21,59 +21,47 @@ To use custom templates and override the theme ones,
 specify the relative path to your templates directory
 with the `custom_templates` global configuration option:
 
-!!! example "mkdocs.yml"
-    ```yaml
-    plugins:
-    - mkdocstrings:
-        custom_templates: templates
-    ```
+```yaml title="mkdocs.yml"
+plugins:
+- mkdocstrings:
+    custom_templates: templates
+```
 
-You directory structure must be identical to the provided templates one:
+Your directory structure must be identical to the provided templates one:
 
 ```
-templates
-├─╴<HANDLER 1>
-│   ├── <THEME 1>
-│   └── <THEME 2>
-└── <HANDLER 2>
-    ├── <THEME 1>
-    └── <THEME 2>
+📁 templates/
+├─╴📁 <HANDLER 1>/
+│   ├── 📁 <THEME 1>/
+│   └── 📁 <THEME 2>/
+└── 📁 <HANDLER 2>/
+    ├── 📁 <THEME 1>/
+    └── 📁 <THEME 2>/
 ```
 
-(*[Check out the template tree on GitHub](https://github.com/mkdocstrings/mkdocstrings/tree/master/src/mkdocstrings/templates/)*)
+For example, check out the Python
+[template tree](https://github.com/mkdocstrings/python/tree/master/src/mkdocstrings_handlers/python/templates/)
+on GitHub.
 
 You don't have to replicate the whole tree,
 only the handlers, themes or templates you want to override.
 For example, to override some templates of the *Material* theme for Python:
 
 ```
-templates
-└── python
-    └── material
-        ├── parameters.html
-        └── exceptions.html
+📁 templates/
+└── 📁 python/
+    └── 📁 material/
+        ├── 📄 parameters.html
+        └── 📄 exceptions.html
 ```
 
 In the HTML files, replace the original contents with your modified version.
 In the future, the templates will use Jinja blocks, so it will be easier
-to modify a small part of the template without copy-pasting the whole file.
-
-The *Material* theme provides the following template structure:
-
-- `children.html`: where the recursion happen, to render all children of an object
-    - `attribute.html`: to render attributes (class-attributes, etc.)
-    - `class.html`: to render classes
-    - `function.html`: to render functions
-    - `method.html`: to render methods
-    - `module.html`: to render modules
-- `docstring.html`: to render docstrings
-    - `attributes.html`: to render attributes sections of docstrings
-    - `examples.html`: to render examples sections of docstrings
-    - `exceptions.html`: to render exceptions/"raises" sections of docstrings
-    - `parameters.html`: to render parameters/arguments sections of docstrings
-    - `return.html`: to render "return" sections of docstrings
-- `properties.html`: to render the properties of an object (`staticmethod`, `read-only`, etc.)
-- `signature.html`: to render functions and methods signatures
+to modify small part of the templates without copy-pasting the whole files.
+
+See the documentation about templates for:
+
+- the Python handler: https://mkdocstrings.github.io/python/customization/#templates
```

Comment:
Let me know if we can link to the Crystal handler's docs as well.

Reply:
Sure, you can link to https://mkdocstrings.github.io/crystal/styling.html